### DISCUSSION TITLE
Will the slicing, postprocess procedure be considered after resize image rather than the original image?

### DISCUSSION THREAD

**Discussion by YoungjaeDev** [obss/sahi#819] [Q&A] [answered]
Hello
**SAHI** I understand that the original image will be used to carry out the subsequent process after slice, but do you have any plans to add a function of predicting after resizing for faster speed? What we need additionally is the part where we have to do the box coordinate as the original (resize box -> origin box).

For example 
3840 x 1080 (Origin) => 1920 x 540 (Resize) => SAHI => Prediction_result => scale_coord(Origin)

**Comment by fcakyon** [comment]
Currently, SAHI works as: 
`(origin)  3840 x 1080 => (slice and resize) 1920 x 540 => SAHI => slice Prediction_result => scale_coord(Origin)`. 

I need help understanding your use-case. Can you be more specific?

**Comment by YoungjaeDev** [comment, marked as answer]
@fcakyon 
https://github.com/obss/sahi/blob/cbe6fd707356137a773654819958ce9f62080a2a/sahi/predict.py#L498

**(origin) 3840 x 1080 => (slice and resize) 1920 x 540** when delivering the image path at the param of predict func, isn't this part applicable?

In other words, isn't it working slice right at 3840 x 1080 (origin size)?

**Reply by fcakyon** [reply]
Yes it slices the original image.
